### PR TITLE
ISPN-13097 Protobuf remote query

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-13097

### CODE REVIEW

File: remote-query/src/main/java/org/infinispan/tutorial/simple/remote/query/InfinispanRemoteQuery.java [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
 import org.infinispan.client.hotrod.RemoteCacheManager;
 import org.infinispan.client.hotrod.Search;
+import org.infinispan.client.hotrod.configuration.ClientIntelligence;
 import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
 import org.infinispan.client.hotrod.impl.ConfigurationProperties;
 import org.infinispan.client.hotrod.marshall.MarshallerUtil;
 import org.infinispan.client.hotrod.marshall.ProtoStreamMarshaller;
 import org.infinispan.commons.api.CacheContainerAdmin;
+import org.infinispan.protostream.GeneratedSchema;
 import org.infinispan.protostream.SerializationContext;
```
Comment (pruivo) [inline remote-query/src/main/java/org/infinispan/tutorial/simple/remote/query/InfinispanRemoteQuery.java:14]: 4 unused commits can be removed :)

File: remote-query/src/main/java/org/infinispan/tutorial/simple/remote/query/Person.java [hunk 2]
```diff
@@ -24,6 +25,7 @@ public Person() {
       // An accessible no-arg constructor is required for Protostream to be able to instantiate this
```
Comment (pruivo) [inline remote-query/src/main/java/org/infinispan/tutorial/simple/remote/query/Person.java:25]: I guess this constructor is not required anymore right?

**Comment by karesti** [comment]
@pruivo rebased

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**